.github/workflows/run-and-test-builds.yml

```diff
@@ -124,4 +124,4 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
-      - run: aws cloudwatch put-metric-data --metric-name RunTimeTestsFailure --namespace GithubCanaryApps --value 0
+      - run: aws cloudwatch put-metric-data --metric-name RunTimeTestsSuccess --namespace GithubCanaryApps --value 0
```

Comment:
We don't have a RunTimeTestsSuccess metric setup with any alarms right now. Is the idea that we'll use this in the future to alarm if it's missing for a certain period of time?

Reply:
From my understand, this line will create a RunTimeTestsSuccess metric and we'll need to manually set up the alarm from the metric. Is that right?

Reply:
Right

Reply:
Looking at this further this might be an issue. The way the alarm works is that if RunTimeTestsFailure > 0 for 2 data points within 40 minutes the alarm is triggered. When a success occurs the `RunTimeTestsFailure` is sent a 0. If a 0 is never sent won't the `RunTimeTestsFailure` be above 0 forever?

Perhaps you have to change the additional configuration to have missing data treated as being good.

<img width="477" alt="image" src="https://user-images.githubusercontent.com/65630/179637024-94e7b372-06cc-4362-b625-210f22337245.png">

Reply:
Or change this back to `RunTimeTestsFailure`

Reply:
With @ErikCH on this one, is this change required? This would break the existing metric because we've been relying on the value 0 and 1 to determine the alarm state.

Reply:
+1

Reply:
Awesome point! Thanks for catching it. That means I need to add the `success` action back to the publish workflow as well.

Just pushed a new commit 😁